### DISCUSSION TITLE
对ack的理解

### DISCUSSION THREAD

**Discussion by novada0** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#68] [Q&A]
请问ack可以表示成与tcp的ack一个意思吗？就是ack3表示表示已经收到了seq1和seq2。但是报告中描述好像是ack3表示收到了seq1、seq2、seq3

**Comment by TANSixu** [comment]
Hi，可以的，这个没有强制要求，能够实现功能就可以啦。只是教材中对于packet-based用的是ack当前收到的，byte-based用的是ack下一个需要的，所以我们packet-based的ack就沿用书上的写法了。
